docs/wiki/The library - Module design.md

```diff
@@ -48,10 +48,12 @@ They can be deployed in different configurations just by changing the input para
   > **Example:** The VirtualNetworkPeering construct leverages the VirtualNetworkPeering module to deploy multiple virtual network peering connections at once.
 - Where the resource type in question supports it, the module should have support for:
   1. **Diagnostic logs** and **metrics** (you can have them sent to one ore more of the following destination types: storage account, log analytics and event hub).
-  2. Resource and child resource level **RBAC** (for example, providing data contributor access on a storage account; granting file share/blob container level access in a storage account)
-  3. **Tags** (as objects)
-  4. **Locks**
-  5. **Private Endpoints** (if supported)
+  1. Resource and child-resource level **RBAC** (for example, providing data contributor access on a storage account; granting file share/blob container level access in a storage account)
+  1. **Tags** (as objects)
+  1. **Locks**
+  1. **Private Endpoints** (if supported)
+  1. **User-defined Types** (if supported)
```

Comment:
🪄

Reply:
I'm confused, is there a todo? :D 🍡

Reply:
Did you mean user-defined types or something else? I felt like they were not intended to go in this list. In general I'd wait for them to be introduced before documenting them as part of the intended design.

Reply:
I think this should be Managed Identities or Customer-Managed Keys :D Will doublecheck. Definitely not User-defined types...

Reply:
@eriqua fixed